src/sentry/eventstore/models.py

```diff
@@ -97,7 +97,7 @@ def datetime(self):
         if column in self._snuba_data:
             return parse_date(self._snuba_data[column]).replace(tzinfo=pytz.utc)
 
-        timestamp = self.data.get("timestamp")
+        timestamp = self.data.get("timestamp") or self.data.get("datetime")
```

Comment:
Where does this come from? Afaik a "datetime" value isn't ever stored in nodedata

Reply:
Found it from here: https://github.com/getsentry/sentry/pull/16558/files#diff-517db614db94ad3b1c1ac6d5396f721fL396

Not sure how else we'd end up with this error.